packages/client/src/pages/CarbonHackCards/RegionRecommendationCard/CloudRegions.ts

```diff
@@ -0,0 +1,176 @@
+  EU_GERMANY: { name: 'germany', options: ['germany'] },
+  EU_GERMANY_NORTH: { name: 'germanynorth', options: ['germanynorth'] },
+  EU_GERMANY_WESTCENTRAL: {
+    name: 'germanywestcentral',
+    options: ['germanywestcentral'],
+  },
+  EU_NORWAY: { name: 'norway', options: ['norway'] },
+  EU_NORWAY_EAST: { name: 'norwayeast', options: ['norwayeast'] },
+  EU_NORWAY_WEST: { name: 'norwaywest', options: ['norwaywest'] },
+  EU_UK: { name: 'uk', options: ['uk'] },
+
+  // Middle East regions
+  ME_UAE: { name: 'uae', options: ['uae'] },
+  ME_UAE_CENTRAL: { name: 'uaecentral', options: ['uaecentral'] },
+  ME_UAE_NORTH: { name: 'uaenorth', options: ['uaenorth'] },
+
+  // America regions
+  US_CANADA: { name: 'canada', options: ['canada'] },
+  US_CANADA_CENTRAL: { name: 'canadacentral', options: ['canadacentral'] },
+  US_CANADA_EAST: { name: 'canadaeast', options: ['canadaeast'] },
+  US_CENTRAL: { name: 'CentralUS', options: ['CentralUS'] },
+  US_CENTRAL_EUAP: { name: 'centraluseuap', options: ['centraluseuap'] },
+  US_CENTRAL_STAGE: { name: 'centralusstage', options: ['centralusstage'] },
+  US_EAST: { name: 'EastUS', options: ['EastUS', 'EASTUS', 'USEast'] },
+  US_EAST_STAGE: { name: 'eastusstage', options: ['eastusstage'] },
+  US_EAST_2: { name: 'EastUS2', options: ['EastUS2', 'useast2'] },
+  US_EAST_2_EUAP: { name: 'eastus2euap', options: ['eastus2euap'] },
+  US_EAST_2_STAGE: { name: 'eastus2stage', options: ['eastus2stage'] },
+  US_EAST_3: { name: 'EastUS3', options: ['EastUS3'] },
+  US_NORTH_CENTRAL: {
+    name: 'NorthCentralUs',
+    options: ['NorthCentralUs', 'northcentralus'],
+  },
+  US_NORTH_CENTRAL_STAGE: {
+    name: 'northcentralusstage',
+    options: ['northcentralusstage'],
+  },
+  US_SOUTH_CENTRAL: { name: 'SouthCentralUS', options: ['SouthCentralUS'] },
+  US_SOUTH_CENTRAL_STAGE: {
+    name: 'southcentralusstage',
+    options: ['southcentralusstage'],
+  },
+  US_US: { name: 'unitedstates', options: ['unitedstates'] },
+  US_US_EAP: { name: 'unitedstateseuap', options: ['unitedstateseuap'] },
+  US_WEST_CENTRAL: {
+    name: 'WestCentralUS',
+    options: ['WestCentralUS', 'westcentralus'],
+  },
+  US_NORTH: { name: 'USNorth', options: ['USNorth'] },
+  US_WEST: { name: 'WestUS', options: ['WestUS', 'westus'] },
+  US_WEST_STAGE: { name: 'westusstage', options: ['westusstage'] },
+  US_WEST_2: { name: 'westus2', options: ['westus2', 'uswest2', 'WESTUS2'] },
+  US_WEST_2_STAGE: { name: 'westus2stage', options: ['westus2stage'] },
+  US_WEST_3: { name: 'westus3', options: ['westus3', 'uswest3'] },
+  BRAZIL: { name: 'brazil', options: ['brazil'] },
+  BRAZIL_SOUTH: { name: 'brazilsouth', options: ['brazilsouth'] },
+  BRAZIL_SOUTH_EAST: { name: 'brazilsoutheast', options: ['brazilsoutheast'] },
+
+  ALL: { name: 'All Regions', options: ['All Regions'] },
+  UNASSIGNED: { name: 'Unassigned', options: ['Unassigned'] },
+  UNKNOWN: { name: 'Unknown', options: ['Unknown'] },
+}
```

Comment:
These regions are already constants within each respective Cloud Provider package and shouldn't be copied over. I think this is another good reason why the SDK logic should live in the API and App packages if it depends on that information from the Cloud Providers